build/Linux+BSD/mscore.1.preview.sh

```diff
@@ -1,4 +1,25 @@
 #!/bin/sh
+##
```

Comment:
I'd say this line above is unnecessary. In the Linux kernel, the identifier always starts straight after the sheband line.